docker/nginx/nginx-x509.conf

```diff
@@ -45,6 +45,15 @@ http {
         listen [::]:80 default_server;
         listen 443 ssl http2 default_server;
         listen [::]:443 ssl http2 default_server;
+
+        set $dynamic_server_name $MG_NGINX_SERVER_NAME;
+
+        if ($dynamic_server_name = '') {
+            set $dynamic_server_name "localhost";
+        }
```

Comment:
If we set it by default to `localhost`, there is no need for this check.

Reply:
If the env variable `$MG_NGINX_SERVER_NAME` is empty, In that case `$dynamic_server_name` should not be empty, So it was set to localhost

Reply:
Changed approach